opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -173,6 +173,8 @@ where
         log_record.set_event_name(meta.name());
         log_record.set_severity_number(severity_of_level(meta.level()));
         log_record.set_severity_text(meta.level().as_str());
+        log_record.set_timestamp(std::time::SystemTime::now());
+        log_record.set_observed_timestamp(std::time::SystemTime::now());
```

Comment:
This is already done in [SDK](https://github.com/open-telemetry/opentelemetry-rust/blob/8bd529a6d629aff7482b875cfc39275a8a71eaeb/opentelemetry-sdk/src/logs/log_emitter.rs#L303), why do we need it here.

Reply:
Is this path in SDK hit when this `Bridge` is added as a layer into `tracing_subscriber` ? On the QuickWit side I see that this timestamp was not populated which needed a workaround: quickwit-oss/quickwit#5366

Reply:
> Is this path in SDK hit when this Bridge is added as a layer into tracing_subscriber

yes it does.